api/agent/pure_runner.go

```diff
@@ -540,6 +546,15 @@ func (pr *pureRunner) handleTryCall(tc *runner.TryCall, state *callHandle) error
 		return err
 	}
 
+	// Just to be safe. Status image is special and exempt from agent scheduler
+	// resource tracker. We need to make sure normal functions calls do not use it
+	// to get around the resource scheduler.
+	if pr.statusImageName != "" && c.Image == pr.statusImageName {
```

Comment:
Does this mean that we're effectively reserving an image by name that the customer will not be able to use?  Is it possible for them to create another image with an identical name and cause a conflict that they won't obviously be able to resolve here?

Reply:
Yes, this has to be a reserved image name (plus tag) that customers cannot use. But operators should be able  to configure a private image name/tag here to ensure such image name clashes do not occur. (eg. 192.0.2.0:9999/fn-status-checker with a private non-routable IP/port prefix)